<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>656</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>true</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Titolo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>26</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>HOME SAFETY</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="aggiungidispositivo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Menu:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="aggTelecamera">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Aggiungi Telecamera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aggSensore">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Aggiungi Sensore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aggAllarme">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Aggiungi Allarme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aggTelAll">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>aggiungi Telecamera Allarmata</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="visualizza">
       <property name="text">
        <string>Visualizza dispositivi totali</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="visualizzaAccesi">
       <property name="text">
        <string>Visualizza dispositivi accesi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elimDispositivo">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Elimina dispositivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="accendi">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Accendi dispositivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spegni">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Spegni Dispositivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="simulaMovimento">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Simula Movimento</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aggiungiDati">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Aggiungi dati fittizzi</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="visDispositivi">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>401</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="messaggio">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="id_elimina">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_accendi">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_spegni">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id_simula">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>184</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID del dispositivo da eliminare:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ID del dispositivo da accendere:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID del dispositivo da spegnere:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ID del dispositivo da simulare:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
